scripts/update/Updater.php

```diff
@@ -31,13 +34,12 @@
  * @author Joel Bout <joel@taotesting.com>
  * @author Jérôme Bogaerts <jerome@taotesting.com>
  */
-class taoResultServer_scripts_update_Updater extends \common_ext_ExtensionUpdater
+class Updater extends common_ext_ExtensionUpdater
 {
 
     /**
-     *
      * @param string $initialVersion
-     * @return string|void
+     * @throws common_Exception
```

Comment:
Since this is implementing an abstraction (or even if it was an interface) you can ignore phpdoc or use only `@inheritDoc` instead. It avoids us to fix phpdoc in multiple places. :)